example/App.tsx

```diff
@@ -209,6 +210,11 @@ const EXAMPLES: ExamplesSection[] = [
         unsupportedPlatforms: new Set(['android', 'ios', 'macos']),
       },
       { name: 'Stylus data', component: StylusData },
+      {
+        name: 'Two finger Pan',
+        component: TwoFingerPan,
+        unsupportedPlatforms: new Set(['android', 'macos', 'ios']),
```

Comment:
```suggestion
        unsupportedPlatforms: new Set(['android', 'macos']),
```

Shouldn't this prop also work on iPads?

Reply:
It should 😅

Reply:
Brought back in [dc59483](https://github.com/software-mansion/react-native-gesture-handler/pull/3163/commits/dc59483274fb14a3478a0e95a3958080a44c5bbd)